addons/info/package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "prepublish": "node ../../scripts/prepublish.js",
     "publish-storybook": "bash .scripts/publish_storybook.sh",
+    "dev": "babel --watch --ignore tests,__tests__,test.js,stories/,story.jsx --plugins transform-runtime ./src --out-dir ./dist --copy-files",
```

Comment:
I don't want to have development script/setup on every individual package in the monorepo.
We want to do development on the entire codebase as a whole.

If you bootstrap the codebase and run `yarn dev` (in the root)
and then start the cra-kitchen-sink example. You should see addons change after a refresh.